PersonStageThree/src/com/beyond/person/helper/SetterClass.java

```diff
@@ -0,0 +1,36 @@
+package com.beyond.person.helper;
+
+import com.beyond.person.core.exception.*;
+import com.beyond.person.core.model.BasePerson;
+
+public class SetterClass {
```

Comment:
@maria-salmasian what is this ? You can't have setter class , describe what you want to solve with this

Reply:
changed the name, added comments